### PR TITLE
Delete unused private method

### PR DESCRIPTION
**Pull request by Youssef1313**

### CODE REVIEW

File: src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs [hunk 1]
```diff
@@ -317,87 +317,6 @@ private static bool IsInAStringOrCharacter(SyntaxNode currentNode, SnapshotPoint
                 && caret.Position < currentNode.Span.End
                 && caret.Position > currentNode.SpanStart;
 
-        private static bool SemicolonIsMissing(SyntaxNode currentNode)
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs:320]: @sharwell i'm not sure how this command handler is supposed to work.  can you take a look?  Seems veryfishythat this isn't being used.

Comment (Youssef1313) [reply src/EditorFeatures/CSharp/CompleteStatement/CompleteStatementCommandHandler.cs:320]: @CyrusNajmabadi I'm unsure too, that's why I opened as draft to see if the CI will fail.
If this is being used, I'd like @sharwell to confirm whether it should be updated with records (it currently have a check for class)

**Comment by Youssef1313** [comment]
@sharwell Could you give a look?

**Review by sharwell — APPROVED** [review]

**Comment by sharwell** [comment]
Likely an oversight from the initial implementation. If we do add it in the future, it would be part of a separate design implementation.

**Comment by Youssef1313** [comment]
@sharwell Should this be ready for merge?

**Comment by Youssef1313** [comment]
Closing since this is being done as part of #52464 (commit https://github.com/dotnet/roslyn/pull/52464/commits/980211990c4d2fabb07ce0bda832a2da934e1752)

**Pull request closed without merging**